lib/metasploit/framework/ssh/platform.rb

```diff
@@ -83,32 +83,32 @@ def self.get_platform_info(ssh_socket, timeout: 10)
         end
 
         def self.get_platform_from_info(info)
-          case info
-          when /unifi\.version|UniFiSecurityGateway/ #Ubiquiti Unifi.  uname -a is left in, so we got to pull before Linux
+          case info.downcase
+          when /unifi\.version|unifisecuritygateway/ #Ubiquiti Unifi.  uname -a is left in, so we got to pull before Linux
```

Comment:
I believe we can use regex flags to ignore the casing here instead:
```suggestion
          case info
          when /unifi\.version|UniFiSecurityGateway/i #Ubiquiti Unifi.  uname -a is left in, so we got to pull before Linux
```